src/jobs/deploy_to_selfhosted.yml

```diff
@@ -11,11 +11,20 @@ steps:
             echo $response
         }
 
-        function get_workflow_status {
-            response=$(curl -s -H 'Accept: application/json' -H 'Content-type: application/json' "https://circleci.com/api/v2/pipeline/$workflowID/workflow" -H "Circle-Token: $CIRCLE_CI_GITHUB_TOKEN" | jq -r '.items[0].status')
+        function get_workflow {
+            response=$(curl -s -H 'Accept: application/json' -H 'Content-type: application/json' "https://circleci.com/api/v2/pipeline/$1/workflow" -H "Circle-Token: $CIRCLE_CI_GITHUB_TOKEN")
             echo $response
         }
 
```

Comment:
I would create another function just to get the status.
```
function get_workflow_status {
    echo $(get_workflow $1 | jq -r '.items[0].status')
}
```

Reply:
added :)